mysql/docker-entrypoint-initdb.d/02_data.sql

```diff
@@ -235,4 +233,64 @@ UPDATE PredictionsFeedback
     SET proposed_label = prediction
 WHERE 1=1
     AND vote = 1
+;
+
+DELIMITER $$
+
+CREATE PROCEDURE InsertAnonymousPlayers(IN NUM INT, IN possible_ban BOOL, IN confirmed_ban BOOL, IN confirmed_player BOOL)
+BEGIN
+    DECLARE i INT DEFAULT 1;
+
+    WHILE i <= NUM DO
+        INSERT INTO Players (
+            name,
+            created_at,
+            updated_at,
+            possible_ban,
+            confirmed_ban,
+            confirmed_player,
+            label_id,
+            label_jagex,
+            ironman,
+            hardcore_ironman,
+            ultimate_ironman,
+            normalized_name
+        )
+        SELECT
+            CONCAT('anonymoususer ', 
+                SUBSTRING(MD5(i), 1, 8), ' ', 
+                SUBSTRING(MD5(i), 9, 4), ' ', 
+                SUBSTRING(MD5(i), 13, 4), ' ', 
+                SUBSTRING(MD5(i), 17, 4), ' ', 
+                SUBSTRING(MD5(i), 21, 12)) AS name, -- anonymous user name
+            NOW() AS created_at, -- updated later
+            NOW() AS updated_at, -- updated later
+            possible_ban,
+            confirmed_ban,
+            confirmed_player,
+            0 AS label_id, 
+            ROUND(RAND() * 1) AS label_jagex, -- doesn't matter?
+            null AS ironman,
+            null AS hardcore_ironman,
+            null AS ultimate_ironman,
+            CONCAT('anonymoususer ', 
+                SUBSTRING(MD5(i), 1, 8), ' ', 
+                SUBSTRING(MD5(i), 9, 4), ' ', 
+                SUBSTRING(MD5(i), 13, 4), ' ', 
+                SUBSTRING(MD5(i), 17, 4), ' ', 
+                SUBSTRING(MD5(i), 21, 12)) AS normalized_name -- anonymous user name
+        FROM dual;
+
+        SET i = i + 1;
+    END WHILE;
+END $$
+
+DELIMITER ;
+
+call InsertAnonymousPlayers(10, 1, 0, 0);

```

Comment:
there is a `UUID()` function, which would mak this cleaner

Reply:
yea i had uuid at first but you cant provide it a 'seed' to ensure the uuids were the same every time.  this was my workaround